vscode/package.json

```diff
@@ -342,6 +342,9 @@
         "extensions": [
           ".qs"
         ],
+        "filenames": [
+          "qsharp.json"
+        ],
```

Comment:
This causes qsharp.json files to be treated as Q# ... we don't want that. I'm getting Q# compiler syntax errors from the json file :)